spec/features/schools/participants/add_participants/sit_add_ect_spec.rb

```diff
@@ -3,7 +3,7 @@
 require "rails_helper"
 require_relative "../../training_dashboard/manage_training_steps"
 
-RSpec.describe "SIT adding themself as ECT", js: true do
+RSpec.describe "SIT adding themself as ECT", js: true, mid_cohort: true do
```

Comment:
The issue was the date we are currently, wasn't it?

Reply:
I'm not sure I understand what you mean. I've used the same helper we've used in other date dependent fixes here.

Reply:
I was just trying to understand and you to confirm that the reason why these test starting to fail was because we are at the beginning of June and the testing cohorts are setup to start registration.
That has affected the tests scenarios basically.

Reply:
Yes I understood this when we added the `mid_cohort` helper to `main` time bomb tests are very common in codebases like this.